api/build.gradle

```diff
@@ -7,6 +7,16 @@ plugins {
     alias(libs.plugins.spring.dependency.management)
 }
 
+configurations.all {
+    resolutionStrategy {
+        capabilitiesResolution {
+            withCapability("org.lz4:lz4-java") {
```

Comment:
Can we add an inline comment/reference pointing to https://seclists.org/oss-sec/2025/q4/222 for this? Otherwise, it looks a bit cryptic